recipe/meta.yaml

```diff
@@ -17,9 +17,10 @@ source:
     - 0001-Force-endian-flag-in-cross-compilation-mode.patch  # [arm64 or aarch64 or ppc64le]
 
 build:
-  number: 2
+  number: 0
   skip: true  # [win and cuda_compiler_version != "None"]
   skip: true  # [cuda_compiler_version == "10.2"]
+  skip: true  # [python_impl == 'pypy' and py < 38]
```

Comment:
Skipped due to the problem identified in this comment ( https://github.com/conda-forge/pypy3.6-feedstock/issues/83#issuecomment-1182510938 ), which also showed up in CI here